components/scream/src/physics/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 add_subdirectory(share)
 add_subdirectory(p3)
-add_subdirectory(rrtmgp)
+if ("${SCREAM_DOUBLE_PRECISION}")
```

Comment:
Minor: the cmake way of testing conditionals is `if (VAR_NAME)` (without dereferencing the variable). Dereferencing the variable could cause a config warning.

Reply:
Ah, cool, thanks! Fixed.